etc/profile-a-l/google-chrome-beta.profile

```diff
@@ -5,11 +5,8 @@ include google-chrome-beta.local
 # Persistent global definitions
 include globals.local
 
-# Disable for now, see https://github.com/netblue30/firejail/pull/3688#issuecomment-718711565
 ignore whitelist /usr/share/mozilla/extensions
 ignore whitelist /usr/share/webext
```

Comment:
I guess google-chrome support web extensions.

```suggestion
```

Reply:
Whether or not google-chrome supports web extensions, the presence of /usr/share/mozilla/extensions and /usr/share/webext shouldn't break google chrome.

Reply:
I deleted

```
ignore whitelist /usr/share/mozilla/extensions
ignore whitelist /usr/share/webext
```